app/assets/javascripts/social_networking/controllers/goal-controller.js

```diff
@@ -11,11 +11,14 @@
     this.studyEndDate = studyEndDate;
     this.noticesEnabled = noticesEnabled;
     this.noticeUtility = noticeUtility;
+    this.displayPopoverHTML = displayPopoverHTML || false;
 
     this.resetForm();
     this.resetTabs();
     if (typeof $ !== 'undefined') {
```

Comment:
Is this test necessary? I know you didn't write it, but it seems pretty pessimistic to assume jQuery didn't get loaded!